src/styles/application/_provisioning.scss

```diff
@@ -0,0 +1,76 @@
+.loadingscreen-backdrop {
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  background-size:cover;
+  background-color: $pf-color-white;
+  background-image: url('../../img/provisioning-bg.png');
+}
+
+.loadingscreen-logo {
+  background-size: cover;
+  height: 36px;
+  width: 113px;
+  position: absolute;
+  top: 10px;
+  right: 50px;
+  background-image: url('../../img/RedHatLogo.png')
+}
+
+.loadingscreen-throbber {
+  display: inline-block;
+  position: fixed;
+  top: 10%;
+  bottom: 50%;
+  left: 0;
+  right: 0;
+  margin: auto;
+}
+
+.loadingscreen-text {
+  display: inline-block;
+  position: fixed;
+  color: $pf-color-white;
+  text-align: center;
+  top: 40%;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 45%;
+  height: 15%;
+  margin: auto;
+}
+
+.loadingscreen-progressbar {
+  display: inline-block;
+  position: fixed;
+  top: 50%;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 50%;
+  margin: auto;
+  background-color: transparent;
+  border: 2px solid $pf-color-white;
+  color: $pf-color-white;
+}
+
+.loadingscreen-progressbar .progress-bar {
+  background-color: $pf-color-white;
+  background-image: none;  
+}
+
+.loadingscreen-progress-label {
+  display: inline-block;
+  position: fixed;
+  top: 55%;
```

Comment:
my bad - this should have been 50% (in order for it to line up with the progress bar).

Reply:
@mindreeper2420 Makes sense, updated